packages/cedar-amcharts/src/render/render.ts

```diff
@@ -98,6 +98,13 @@ export function fillInSpec(spec: any, definition: any) {
 }
 
 export function fetchSpec(type: string): any {
+  if (type === 'time') {
+    console.warn(`'Time' is no longer a supported type. Please use 'line' instead`)
+    return clone(specs['line'])
+  } else if (type === 'bubble') {
+    console.warn(`'Bubble' is no longer a supported type. Please use 'scatter' instead`)
+    return clone(specs['scatter'])
+  }
   return clone(specs[type])
```

Comment:
Let's switch over to the new `clone()` in this PR, see https://github.com/Esri/cedar/issues/350#issuecomment-351228006

Reply:
nvm, still need deep clone